src/js/user-profile/components/UserDataSection.jsx

```diff
@@ -69,13 +87,36 @@ class UserDataSection extends React.Component {
     );
   }
 
+  renderMultifactorMessage() {
```

Comment:
Should this check for `this.props.profile.multifactor`?

Reply:
Fixed. Also added some fixes for the E2E tests